pkgs/applications/networking/irc/konversation/1.6.nix

```diff
@@ -0,0 +1,65 @@
+{ stdenv, lib, fetchurl, cmake, kde5, makeQtWrapper, phonon}:
+
+let
+  pn = "konversation";
+  v = "1.6";
+
+  buildInputs = with kde5; [
+    cmake
+    extra-cmake-modules
+    kbookmarks
+    karchive
+    kconfig
+    kconfigwidgets
+    kcoreaddons
+    kdbusaddons
+    kdoctools
+    kemoticons
+    kglobalaccel
+    ki18n
+    kiconthemes
+    kidletime
+    kitemviews
+    knotifications
+    knotifyconfig
+    kio
+    kparts
+    kwallet
+    solid
+    sonnet
+    phonon
+  ];
+in
+
+stdenv.mkDerivation rec {
+  name = "${pn}-${v}";
+
+  src = fetchurl {
+    url = "mirror://kde/stable/${pn}/${v}/src/${name}.tar.xz";
+    sha256 = "789fd75644bf54606778971310433dbe2bc01ac0917b34bc4e8cac88e204d5b6";
+  };
+
+  nativeBuildInputs = [
+    kde5.extra-cmake-modules
+    kde5.kdoctools
+    makeQtWrapper
+  ];
+
+  inherit buildInputs;
+
+  cmakeFlags =
+    (buildInputs.cmakeFlags or [])
+    ++ [ "-DBUILD_TESTING=OFF" ];
+    #++ lib.optional debug "-DCMAKE_BUILD_TYPE=Debug";
+
+  postInstall = ''
+    wrapQtProgram "$out/bin/konversation"
+  '';
+
+  meta = with stdenv.lib; {
+    description = "Integrated IRC client for KDE";
+    repositories.git = git://anongit.kde.org/konversation;
+    license = "GPL";
+    inherit (kdelibs.meta) maintainers platforms;
```

Comment:
This shouldn't inherit from `kdelibs` because it doesn't depend on it. I also don't think the `kdelibs` maintainers want to take over this package! Add yourself as a maintainer instead.